apps/condo/domains/property/components/BasePropertyForm/index.tsx

```diff
@@ -164,7 +164,7 @@ const BasePropertyForm: React.FC<IPropertyFormProps> = (props) => {
                                             setAddressValidatorError={setAddressValidatorError}
                                             onSelect={(_, option) => {
                                                 const address = JSON.parse(option.key as string) as AddressMetaField
-                                                if (!validHouseTypes.includes(address.data.house_type_full)) {
+                                                if (!validHouseTypes.includes(address.data.house_type_full) && address.data.stead_type_full == null) {
```

Comment:
Why not use get? 

Also i'd move logic with stead to different place, like with !validHouseTypes

Reply:
Looks something wrong here